### PR TITLE
Barozine fix

### PR DESCRIPTION
**Pull request by SX-7**
## About the PR
Barozine now accumulates. Also few tweaks to its status effects.

## Why / Balance
Previously barozine didn't acumulate (even though it should), resulting in very low pressure immunity duration - now it does accumulate. Also, the recipe for crafting it now uses plasma as catalyst instead, which should make it easier to craft for chemists, and less of a headache for yowies to request it from chemistry in non-zero quantities.

That being said, since both changes result in a powerspike that can be harnessed by everyone, not just yowies that usually require it round-end, two nerfs were added. 
- The heat self-damage was increased to guaranteed 0.25, which on average almost doubles it (spacepen now deals 10 damage, guaranteed).
- It slows you down by 10%, which is equivalent to lighter end of hardsuits (bigger change on paper than in practice, mainly to dissuade people using it instead of spacesuits for combat purposes)

## Technical details
Basically everything is a trivial yaml change. Barozine yaml didn't use `refresh: false` before, now it does.

## Requirements
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

**Changelog**
:cl:
- tweak: Barozine is easier to make in larger quantities, but has more downsides (more self-damage and slowdown).
- fix: Barozine effect now accumulates.

### CODE REVIEW

**Comment by BombasterDS2** [comment]
Accumulating is overpowered for barozine. It increases space medipen pressure immunity from 22 seconds to 80 seconds. With plasma working as catalyst it makes barozine super cheap to produce.

MovementSpeed debuff will work 20 seconds only so character will be immune to space for 60 seconds without any speed debuff.

**Comment by BombasterDS2** [comment]
Since there is no OD for barozine you'll be able to make 50:10 barozine:pyrazine to get 200 seconds of pressure immunity that will allow you to wear good armor without thinking about spacing.

**Comment by SX-7** [comment]
Oh, latter is a good nitpick, slipped by me. Will fix 👍

As for former, it's kinda hard not to say "And...?" You can get permanent spacefaring capabilities with literally any hard suit, and you also get damage resistances. I'd have to check the entire list, but if it's better than emergency eva suit and not much else, then I don't see it being a major issue.

I do agree that space medipen having 80 seconds is kinda too good.

**Comment by SX-7** [comment]
> Since there is no OD for barozine you'll be able to make 50:10 barozine:pyrazine to get 200 seconds of pressure immunity that will allow you to wear good armor without thinking about spacing that makes barozine one of the best fighting chems (barozine shouldn't work like that)

Fair point, upping the OD downsides would help, especially the vomiting (since it cleanses the bloodstream)

**Comment by SX-7** [comment]
Thanks to your feedback (and a walk 😄), I think I figured out a better way to solve that "Yowie issue" without touching chemicals. Will require a bit more work, but I believe that something like allowing Yowies to wear EVA at a massive movement speed penalty would be a less disruptive solution. The mismatch between what's happening and what's described in the guidebook will be resolved in another PR instead.
